targets/UnrealMarketplacePlugin/source/PlayFab/Source/PlayFabCpp/Private/PlayFab.cpp.ejs

```diff
@@ -0,0 +1,53 @@
+<%- copyright %>
+
+  #include "PlayFab.h"
+
+  #include "PlayFabSettings.h"
+
+// Api's
+  <% for(var i = 0; i < apis.length; i++) { var api = apis[i];
+%>#include "Core/PlayFab<%- api.name %>API.h"
+<% } %>
+DEFINE_LOG_CATEGORY(LogPlayFab);
+
+class FPlayFabModule : public IPlayFabModuleInterface
+{
+    /** IModuleInterface implementation */
+    virtual void StartupModule() override;
+    virtual void ShutdownModule() override;
+
+    FString GetTitleId() const override { return PlayFab::PlayFabSettings::GetTitleId(); };
+    
+<% for(var i = 0; i < apis.length; i++) { var api = apis[i];
+%>    PlayFab<%- api.name %>Ptr Get<%- api.name %>API() const override { return <%- api.name %>API; };
+<% } %>
+<% for(var i = 0; i < apis.length; i++) { var api = apis[i];
+%>    PlayFab<%- api.name %>Ptr <%- api.name %>API;
+<% } %>
+
+};
+
+void FPlayFabModule::StartupModule()
+{
+    // create the API
+    ClientAPI = MakeShareable(new PlayFab::UPlayFabClientAPI());
+    AuthenticationAPI = MakeShareable(new PlayFab::UPlayFabAuthenticationAPI());
+    CloudScriptAPI = MakeShareable(new PlayFab::UPlayFabCloudScriptAPI());
+    DataAPI = MakeShareable(new PlayFab::UPlayFabDataAPI());
+    EventsAPI = MakeShareable(new PlayFab::UPlayFabEventsAPI());
+    GroupsAPI = MakeShareable(new PlayFab::UPlayFabGroupsAPI());
+    ProfilesAPI = MakeShareable(new PlayFab::UPlayFabProfilesAPI());
+
+#if WITH_SERVER_CODE
```

Comment:
This section is not properly templatized.
Check this out:
https://github.com/PlayFab/SDKGenerator/blob/33bfebcc79bafc5f50e511b40fe38b7bef0f0a0a/targets/cpp-unreal/PluginSource/Source/PlayFab/Private/PlayFab.cpp.ejs#L23

Reply:
fixed